huntsman/utils/sshfs_mount.py

```diff
@@ -26,9 +32,13 @@ def mount(mountpoint, remote, server_alive_interval=20,
     except FileExistsError:
         pass #For some reason this is necessary
         
+    #SSH options
+    strict_host_key_checking = "yes" if strict_host_key_checking else "no"
     options = f'ServerAliveInterval={server_alive_interval},' + \
-              f'ServerAliveCountMax={server_alive_count_max}'
+              f'ServerAliveCountMax={server_alive_count_max},' + \
+              f'StrictHostKeyChecking={strict_host_key_checking}'
     options = ['sshfs', remote, mountpoint, '-o', options]
+    
     try:
         subprocess.run(options, shell=False, check=True)
```

Comment:
What happens if the ssh link falls over? Will entire docker image crash or will it wait until it reappears?

Reply:
It is configured to retry the connection periodically, although this expires after a few minutes in which case it will cause problems. It can probably (easily) be made a bit more robust in future.